SwrveSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java

```diff
@@ -296,6 +330,154 @@ public SwrveConfigBase setDbName(String dbName) {
     }
 
     /**
+     * Based on the stack that has been chosen, return the correct prefix
+     * @return
+     */
+    private String getStackPrefix(){
+        String stackPrefix = "";
+
+        switch (stack){
+            case US:
+                // No prefixes
+                break;
+            case EU:
+                stackPrefix= "eu.";
+                break;
+            case ACTIVISION:
+                stackPrefix = "activision.";
+                break;
+            case FEATURESTACK:
+                stackPrefix = "featurestack" + Integer.toString(getFeatureStackNum()) + "-";
+                break;
+        }
+        return stackPrefix;
+    }
+
+    /**
+     * Unless the user/developer has already set the content or event url, regenerate the urls based on config information.
+     *
+     * @return
+     * @throws MalformedURLException
+     */
+    private SwrveConfigBase regenerateURLS() {
```

Comment:
why the regenerateURLS? Why not just generateURLS? Same for regenerateUrlFor below.....